staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2874,7 +2874,21 @@ type InterfaceBindingPlugin struct {
 	// ComputeResourceOverhead specifies the resource overhead that should be added to the compute container when using the binding.
 	// version: v1alphav1
 	// +optional
-	ComputeResourceOverhead *k8sv1.ResourceRequirements `json:"computeResourceOverhead,omitempty"`
+	ComputeResourceOverhead *ResourceRequirementsWithoutClaims `json:"computeResourceOverhead,omitempty"`
+}
+
+// ResourceRequirementsWithoutClaims describes the compute resource requirements.
+type ResourceRequirementsWithoutClaims struct {
```

Comment:
Please consider adding a comment that this type is a "fork" of a K8s type.

Reply:
Sure! Will do

Reply:
Thank you.